tests/__init__.py

```diff
@@ -26,68 +31,109 @@
+    test_data_dirs = []
+    test_data_exists = True
+    for p in sessions_dirs:
+        try:
+            test_data_dirs.append(find_full_path(get_ephys_root_data_dir(), p))
+        except FileNotFoundError:
+            test_data_exists = False   # If data not found
 
-    if not test_data_exists:
+    if not test_data_exists:           # attempt to djArchive dowload
         try:
             dj.config['custom'].update({
-                'djarchive.client.endpoint': os.environ['DJARCHIVE_CLIENT_ENDPOINT'],
-                'djarchive.client.bucket': os.environ['DJARCHIVE_CLIENT_BUCKET'],
-                'djarchive.client.access_key': os.environ['DJARCHIVE_CLIENT_ACCESSKEY'],
-                'djarchive.client.secret_key': os.environ['DJARCHIVE_CLIENT_SECRETKEY']
+                'djarchive.client.endpoint':
+                    os.environ['DJARCHIVE_CLIENT_ENDPOINT'],
+                'djarchive.client.bucket':
+                    os.environ['DJARCHIVE_CLIENT_BUCKET'],
+                'djarchive.client.access_key':
+                    os.environ['DJARCHIVE_CLIENT_ACCESSKEY'],
+                'djarchive.client.secret_key':
+                    os.environ['DJARCHIVE_CLIENT_SECRETKEY']
             })
         except KeyError as e:
             raise FileNotFoundError(
-                f'Test data not available at {test_data_dir}.'
+                f' Full test data not available.'
                 f'\nAttempting to download from DJArchive,'
                 f' but no credentials found in environment variables.'
                 f'\nError: {str(e)}')
 
         import djarchive_client
         client = djarchive_client.client()
-        workflow_version = workflow_array_ephys.version.__version__
+        workflow_version = 'v2'
+
+        test_data_dir = get_ephys_root_data_dir()
+        if isinstance(test_data_dir, list):  # if multiple root dirs, first
+            test_data_dir = test_data_dir[0]
 
-        client.download('workflow-array-ephys-test-set',
-                        workflow_version.replace('.', '_'),
+        client.download('workflow-array-ephys-benchmark',
+                        'v2',
                         str(test_data_dir), create_target=False)
     return
 
 
 @pytest.fixture
 def pipeline():
-    from workflow_array_ephys import pipeline
+    if verbose:
+        from workflow_array_ephys import pipeline
 
-    yield {'subject': pipeline.subject,
-           'lab': pipeline.lab,
-           'ephys': pipeline.ephys,
-           'probe': pipeline.probe,
-           'session': pipeline.session,
-           'get_ephys_root_data_dir': pipeline.get_ephys_root_data_dir}
+        yield {'subject': pipeline.subject,
+               'lab': pipeline.lab,
+               'ephys': pipeline.ephys,
+               'probe': pipeline.probe,
+               'session': pipeline.session,
+               'get_ephys_root_data_dir': pipeline.get_ephys_root_data_dir}
```

Comment:
What output does this block create?  Wondering if we can refactor this block out of the if/else statement.

Reply:
The duplication currently under the `pipeline` fixture is motivated the verbosity option I added. If not verbose, the pipeline is constructed under an instance of the `QuietStdOut` class. I was motivated to add a verbosity flag after running the pytests several times and having difficulty visually parsing the errors. 

There are probably other ways to quiet standard output, but this seemed to be the cleanest of the options I found. I would welcome a proposed alternative

Reply:
@kabilar - yes, you were right. functionality was the same if the verbose conditional was only applied to the teardown